### ISSUE TITLE
[SDK 49][expo-image][Android] tintColor fails with SVG images

### ISSUE THREAD

**Issue by masonle2x2** [expo/expo#24511]
### Minimal reproducible example

See my code below

### Summary

tintColor doesn't work with svg images on Android

```tsx
 <Image
        style={{width: 24, height: 24}}
        tintColor={'red'}
        contentFit={'contain'}
        source={required('./tick.svg')}
      />
```
<img width="327" alt="image" src="https://github.com/expo/expo/assets/43195241/fa89ef00-d166-479a-ad37-44556f5c475a">

### Environment

  expo-env-info 1.0.5 environment info:
    System:
      OS: macOS 13.4.1
      Shell: 5.9 - /bin/zsh
    Binaries:
      Node: 20.6.1 - ~/Library/Caches/fnm_multishells/30433_1695091376474/bin/node
      Yarn: 1.22.19 - ~/Library/Caches/fnm_multishells/30433_1695091376474/bin/yarn
      npm: 9.8.1 - ~/Library/Caches/fnm_multishells/30433_1695091376474/bin/npm
      Watchman: 2023.09.04.00 - /opt/homebrew/bin/watchman
    Managers:
      CocoaPods: 1.12.1 - /opt/homebrew/bin/pod
    SDKs:
      iOS SDK:
        Platforms: DriverKit 22.4, iOS 16.4, macOS 13.3, tvOS 16.4, watchOS 9.4
      Android SDK:
        Android NDK: 22.1.7171670
    IDEs:
      Android Studio: 2022.3 AI-223.8836.35.2231.10406996
      Xcode: 14.3.1/14E300c - /usr/bin/xcodebuild
    npmPackages:
      expo: ^49.0.7 => 49.0.10 
      react: 18.2.0 => 18.2.0 
      react-native: 0.72.4 => 0.72.4 
    Expo Workflow: bare

**Comment by kadikraman** [comment]
Hi @MasonLe2497 please can you provide a minimal reproducible example that we could easily run to verify this issue. A code snippet is not enough as it does not include details about the code environment nor the svg.
A minimal reproducible example should be e.g. a link to a Snack or a code repo hosted in GitHub.

**Comment by MasonLe2497** [comment]
Hi @kadikraman , can u check in [repo](https://github.com/MasonLe2497/expo-image-svg). Then can we re-open this issue?

**Comment by 7dp** [comment]
Hi @kadikraman Thanks a lot for the fix. May I know the minimum Expo version to use this fix? (As far as  I know this fix is released at expo-image version 1.6.0)
